Consul/Configuration.cs

```diff
@@ -1011,8 +1011,9 @@ public class ServiceResolverEntry : IConfigurationEntry
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public Dictionary<string, string> Meta { get; set; }
 
+        [JsonConverter(typeof(DurationTimespanConverter))]
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
-        public string ConnectTimeout { get; set; }
+        public TimeSpan? ConnectTimeout { get; set; }
```

Comment:
It breaks backward compatibility, but it is ok to do here. `Configuration` is a fresh addition and I anticipate it is not heavily used yet.

Reply:
Okey, sounds good, I was going off of existing code and changed it for consistency